test/parallel/test-util-decorate-error-stack.js

```diff
@@ -26,7 +26,7 @@ function checkStack(stack) {
 }
 let err;
 const badSyntaxPath =
-    path.resolve(__dirname, '..', 'fixtures', 'syntax', 'bad_syntax')
+    path.resolve(common.fixturesDir, 'syntax', 'bad_syntax')
```

Comment:
I'm going to quit commenting on the same thing, but I think `resolve()` can probably be dropped throughout now.